jake-solution.js

```diff
@@ -0,0 +1,8 @@
+var num = 0;
```

Comment:
Tomorrow morning, could you move all this into a function, called `getSum`, that returns the sum of numbers 3 to 5? I should be able to print out the sum by running:
```
console.log(getSum());
```